docs/pages/build/advanced-credentials-configuration.md

```diff
@@ -33,9 +33,24 @@ The `credentials.json` file should be located at the root of your project, next
 It specifies credentials for Android and iOS (but you can configure only one of these platforms if you don't want to build for the other).
 **Remember to add `credentials.json` to `.gitignore` so you won't accidentally commit it to the repository (and therefore leak your secrets)**.
 
-## Android Credentials
+## Configuring credentials.json
 
-### Obtaining the Keystore
+You can either let Expo generate new credentials and store them on Expo servers or create them manually and put them in credentials.json. Whichever option you choose you can always sync credentials one way or the other with `expo eas:credentials:sync`.
```

Comment:
```suggestion
You have two options for handling your app credentials:
- Let Expo generate and manage your credentials for you, and keep them stored securely on Expo's servers, or
- Create credentials manually, yourself, and provide them to EAS Build via the credentials.json file

No matter which option you choose, you can always sync your credentials by running `expo eas:credentials:sync`, so that the ones you have stored locally match those on Expo's servers.
```